### PR TITLE
[TRANSLATION] add indonesian language translation

### PR DESCRIPTION
**Pull request by Mas4hmad**
## Purpose

to add and update several apps and shell translation to indonesian language

### CODE REVIEW

**Comment by katahiromz** [comment]
```txt
i686-w64-mingw32-windres: /home/travis/build/reactos/reactos/dll/win32/comdlg32/lang/cdlg_Id.rc:22: syntax error
```
The translation of `comdlg32` has errors. Fix please.

File: dll/win32/comdlg32/lang/cdlg_Id.rc [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+LANGUAGE LANG_Indonesian, SUBLANG_DEFAULT
```
Comment (katahiromz) [inline dll/win32/comdlg32/lang/cdlg_Id.rc]:
```suggestion
LANGUAGE LANG_INDONESIAN, SUBLANG_DEFAULT
```
Make it uppercase.

File: dll/win32/themeui/themeui.rc [hunk 2]
```diff
@@ -25,6 +25,9 @@ LANGUAGE LANG_NEUTRAL, SUBLANG_NEUTRAL
 #ifdef LANGUAGE_ES_ES
     #include "lang/es-ES.rc"
 #endif
+#ifdef LANGUAGE_ID_ID
+    #include "lang/id-ID.rc"
```
Comment (katahiromz) [inline dll/win32/themeui/themeui.rc]: You needs `dll/win32/themeui/lang/id-ID.rc`.

File: dll\win32\themeui\lang\id-ID [hunk 3]
```diff
@@ -0,0 +1,35 @@
+LANGUAGE LANG_INDONESIAN, SUBLANG_DEFAULT
```
Comment (katahiromz) [inline dll\win32\themeui\lang\id-ID]: The resource filename extension must be `.rc`.

**Comment by HBelusca** [comment]
@Mas4hmad , why closing your PR and deleting your branch without any explanation?

**Comment by Mas4hmad** [comment]
apologize for inconveniences.
because my github branch is not synced with local branch, and i don't know to repair it. so i will make a PR again at the moment.

**Comment by Mas4hmad** [comment]
i'm sorry, but how to clean all commit up? so i will commit again in this PR.
Thanks before

**Comment by HBelusca** [comment]
You can always add new commits, and then move them around or squash them together. For this you should look at the `git rebase` command in interactive mode.

File: base/applications/shutdown/lang/id-ID.rc [hunk 4]
```diff
@@ -0,0 +1,103 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```
Comment (katahiromz) [inline base/applications/shutdown/lang/id-ID.rc]: Not English.

Comment (katahiromz) [inline base/applications/shutdown/lang/id-ID.rc]:
```suggestion
LANGUAGE LANG_INDONESIAN, SUBLANG_DEFAULT
```

Comment (Mas4hmad) [reply base/applications/shutdown/lang/id-ID.rc]: thanks

**Comment by Mas4hmad** [comment]
sorry again, i was mistake about force pushing... i'm afraid have a wrong commit to master

**Comment by learn-more** [comment]
Don't be afraid of breaking our master, you cannot do that.

**Comment by learn-more** [comment]
Don't be afraid of breaking our master, you cannot do that.

**Comment by Mas4hmad** [comment]
> 
> 
> Don't be afraid of breaking our master, you cannot do that.

yes, but how do i do to force push them to upstream?

**Comment by learn-more** [comment]
You can't force push to `reactos:master`, but you can force push to `Mas4hmad:indonesian-translation`

**Comment by katahiromz** [comment]
@Mas4hmad Are you ready to go?

**Comment by HBelusca** [comment]
I'm handling this one.

**Comment by Mas4hmad** [comment]
Yes, please... Sorry for inconviences and thanks.

**Pull request merged**